qa/tasks/cephfs/filesystem.py

```diff
@@ -172,8 +172,10 @@ def clear_ceph_conf(self, subsys, key):
         del self._ctx.ceph['ceph'].conf[subsys][key]
         write_conf(self._ctx)
 
-    def json_asok(self, command, service_type, service_id):
-        proc = self.mon_manager.admin_socket(service_type, service_id, command)
+    def json_asok(self, command, service_type, service_id, timeout=None):
+        if timeout is None:
+            timeout = 15*60
```

Comment:
Wow, 15m to time out a query to the admin socket? That seems pretty long, but hopefully this is just being conservative.

Reply:
This is just a default upper bound. Some commands like journal flush can take some time so 15 minutes is just being a bit conservative.